app/views/articles/show.html.erb

```diff
@@ -193,7 +193,7 @@
       </article>
 
       <div class="pb-4 crayons-layout__comments-display-ad">
-        <div class="js-display-ad-comments-container pb-4 crayons-layout__comments-display-ad" data-async-url="<%= "/#{@article.username}/#{@article.slug}/display_ads/post_comments" %>"></div>
+        <div class="js-display-ad-container pb-4 crayons-layout__comments-display-ad" data-async-url="<%= "/#{@article.username}/#{@article.slug}/display_ads/post_comments" %>"></div>
```

Comment:
With the same caveat, this could potentially become:

```suggestion
        <div class="js-display-ad-container pb-4 crayons-layout__comments-display-ad" data-async-url="<%= article_display_ad_path placement_area: :post_comments %>"></div>
```

Reply:
I think generating the URL with explicit params is a good option, but I wonder, were the failing specs view specs maybe? (View specs don't automatically set up params.)

Reply:
Yup it was view specs 

<img width="1094" alt="Screenshot 2023-04-13 at 11 41 26" src="https://user-images.githubusercontent.com/2786819/231764207-c91e69a1-babf-40db-a6bc-e83759f69238.png">